Example/Backpack/ViewControllers/IconsViewController.swift

```diff
@@ -25,17 +25,28 @@ class IconsViewController: UICollectionViewController {
     fileprivate static var largeIconList = iconList.filter({ !$0.hasSuffix("-sm") })
     fileprivate static var smallIconList = iconList.filter({ $0.hasSuffix("-sm") })
 
+    enum IconType {
+        case small
+        case large
+    }
+
```

Comment:
@k0nserv I don't think this is adding any value over BPKIconSize as we're just switching on it anyway?